evaluation_results/evaluation_results.csv

```diff
@@ -0,0 +1,4 @@
+Metric,Score
+EM,90.0
+Schema_Correct,100.0
+Levenshtein_Distance,2.3

```

Comment:
The results need to have a description that contains all the information to reproduce the results including the generation of the model.

Reply:
I added the description.